refactor(cli): remove show requests and show results (#1049)
IBM/ado: .cursor/skills/using-ado-cli/SKILL.md

# Get a summary of what has been sampled from the space
uv run ado show details space SPACE_ID

> # Inspect request/result trace for an operation

Reviewer: ```suggestion
# Inspect the trace of measurement requests for an operation
```

<!-- markdownlint-disable line-length -->

| Command                       | What It Shows                                                          |
| ----------------------------- | ---------------------------------------------------------------------- |
| `show measurements operation` | Entities (inputs) and their measurements (outputs) from this operation |
| `show measurements space`     | All entities and measurements collected in this space                  |

> | `show trace operation`        | Measurement requests and results trace for an operation (metadata)     |

Reviewer: ```suggestion
| `show trace operation`        | The trace of measurement requests made during an explore operation. Optionally can show per entity measurement metadata    |
```

<!-- markdownlint-enable line-length -->

```bash
# Measurement data
uv run ado show measurements operation op-123

> # Request/result trace (metadata)

Reviewer: ```suggestion
# Inspect the trace of measurement requests for an operation
```
